app/src/main/res/layout/postitem.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="150dp"
+    android:layout_margin="4dp"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <com.example.sharewhatyoucanproject.imgviewstyling.RoundRectCornerImageView
+            android:id="@+id/postimg"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:adjustViewBounds="true"
+            android:background="@drawable/curvebtn"
+            android:clipToOutline="true"
+            android:scaleType="centerCrop"
+            android:src="@drawable/pizza" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:orientation="vertical"
+            android:paddingVertical="4dp"
+            android:paddingHorizontal="4dp"
+            android:background="@drawable/textcorner"
+            android:layout_gravity="bottom"
+            android:layout_height="wrap_content">
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:maxLines="1"
+                android:text="My title"
+                android:textSize="12sp"
+                android:id="@+id/posttitle"
+                android:textColor="@color/white"
+                android:textStyle="bold"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:textSize="12dp"
+                android:layout_marginTop="4dp"
+                android:id="@+id/postdesc"
+                android:layout_height="wrap_content"
+                android:text="hello \nthis is vegitable pizza \nHome made one"
```

Comment:
you can use `tools:text` instead of `android:text` to add a test value for the previewer.

Reply:
done